index.js

```diff
@@ -12,92 +11,155 @@ console.log(fifaData);
 (d) Away Team goals for 2014 world cup final
 (e) Winner of 2014 world cup final */
 
+let home2014 = fifaData.filter((game) => {
+  if (game["Year"] === 2014 && game["Stage"] === "Final") {
+    console.log(game["Home Team Name"]);
+  }
+});
+
+let away2014 = fifaData.filter((game) => {
+  if (game["Year"] === 2014 && game["Stage"] === "Final") {
+    console.log(game["Away Team Name"]);
+  }
+});
+
+let homeGoals2014 = fifaData.filter((game) => {
+  if (game["Year"] === 2014 && game["Stage"] === "Final") {
+    console.log(game["Home Team Goals"]);
+  }
+});
+
+let awayGoals2014 = fifaData.filter((game) => {
+  if (game["Year"] === 2014 && game["Stage"] === "Final") {
+    console.log(game["Away Team Goals"]);
+  }
+});
 
 /* Task 2: Create a function called  getFinals that takes `data` as an argument and returns an array of objects with only finals data */
 
-function getFinals(/* code here */) {
-
-    /* code here */
-
-};
+function getFinals(data) {
+  let newArray = data.filter((game) => game.Stage === "Final");
+  return newArray;
+}
+console.log(getFinals(fifaData));
 
 /* Task 3: Implement a higher-order function called `getYears` that accepts the callback function `getFinals`, and returns an array called `years` containing all of the years in the dataset */
 
-function getYears(/* code here */) {
-
-    /* code here */
-
-};
-
-getYears();
-
-/* Task 5: Implement a higher-order function called `getWinners`, that accepts the callback function `getFinals()` and determine the winner (home or away) of each `finals` game. Return the name of all winning countries in an array called `winners` */ 
-
-function getWinners(/* code here */) {
-
-    /* code here */
+function getYears(years) {
+  let yearsArray = years.map((game) => game.Year);
+  return yearsArray;
+}
+console.log(getYears(getFinals(fifaData)));
 
-};
+/* Task 4: Implement a higher-order function called `getWinners`, that accepts the callback function `getFinals()` and determine the winner (home or away) of each `finals` game. Return the name of all winning countries in an array called `winners` */
 
-getWinners();
+function getWinners(winner) {
+  let winners = winner.map((game) => {
+    if (game["Home Team Goals"] > game["Away Team Goals"]) {
+      return game["Home Team Name"];
+    } else {
```

Comment:
if else statements are super fun, but can be super confusing sometimes aha (: Great work!

Not sure if you've looked into them too much, but terinarys are similar to if/else statements and pretty cool to play around with to! 

awesome work though!